package.json

```diff
@@ -37,5 +39,24 @@
                 "path": "./syntaxes/alloy.tmLanguage.json"
             }
         ]
+    },
+    "scripts": {
+        "vscode:prepublish": "npm run compile",
+        "compile": "tsc -p ./",
+        "watch": "tsc -watch -p ./",
+        "pretest": "npm run compile && npm run lint",
+        "lint": "eslint src --ext ts",
+        "test": "vscode-test"
+    },
+    "devDependencies": {
+        "@types/vscode": "^1.91.0",
```

Comment:
It seems this needs to line up with the vscode engine version on line 14; `vsce package` reports that `@types/vscode`'s version can't be greater than the engine version.